openmc/model/model.py

```diff
@@ -1142,38 +1162,63 @@ def differentiate_depletable_mats(self, diff_volume_method: str):
             volume equally between the new materials, 'match cell' sets the
             volume of the material to volume of the cell they fill.
         """
+        self.differentiate_mats(diff_volume_method, depletable_only=True)
+
+    def differentiate_mats(self, diff_volume_method: str = None, depletable_only: bool = True):
+        """Assign distribmats for each depletable material
+
+        .. versionadded:: 0.14.0
+
+        Parameters
+        ----------
+        diff_volume_method : str
+            Specifies how the volumes of the new materials should be found.
+            Default is to 'divide equally' which divides the original material
+            volume equally between the new materials, 'match cell' sets the
+            volume of the material to volume of the cell they fill.
+        """
+        if diff_volume_method not in ("divide equally", "match cell", None):
```

Comment:
Let's use the `check_value` function here to verify the volume method. It will need to be included at the top of this file:

```python
from .checkvalue import check_value
```